core/variant.h

```diff
@@ -136,7 +136,7 @@ class Variant {
 		int64_t _int;
 		double _real;
 		Transform2D *_transform2d;
-		Rect3 *_rect3;
+		::AABB *_aabb;
```

Comment:
I wonder if the Variant enum was not named `_AABB` in the past to avoid this naming collision. But maybe this solution is better.

Reply:
I agree the solution is better

Reply:
Wow it's good as-is then?

Reply:
Couldn't be sure myself